app/audit/models/ClaimARefundAuditModel.scala

```diff
@@ -43,17 +43,16 @@ case class ClaimARefundAuditModel(balanceDetails: Option[BalanceDetails],
   }
 
   private def getCreditType(credit: (DocumentDetailWithDueDate, FinancialDetail)): String = {
-    val isMFA: Boolean = credit._2.isMFACredit
-    val isCutOverCredit: Boolean = credit._2.isCutOverCredit
+    val creditType: Option[String] = credit._2.getCreditType
     val isPayment: Boolean = credit._1.documentDetail.paymentLot.isDefined
-    val isBalancingCreditCharge: Boolean = credit._2.isBalancingChargeCredit
-
-    (isMFA, isCutOverCredit, isBalancingCreditCharge, isPayment) match {
-      case (true, _, _, _) => "Credit from HMRC adjustment"
-      case (_, true, _, _) => "Credit from an earlier tax year"
-      case (_, _, true, _) => "Balancing charge credit"
-      case (_, _, _, true) => s"Payment made on ${getFullDueDate(credit._1.dueDate.get)}"
-
+    (creditType, isPayment) match {
```

Comment:
I would really think that isPayment ise in this pattern matching not required, as its really used once